demo/SIMPLE-SPEC.statement.1/kcfg/kcfg.json

```diff
@@ -0,0 +1 @@
+{"next": 4, "nodes": [1, 2, 3], "edges": [{"source": 1, "target": 3, "depth": 1}], "covers": [{"source": 3, "target": 2, "csubst": {"subst": {"GENERATEDCOUNTER_CELL_5e3e01ba": {"node": "KVariable", "name": "GENERATEDCOUNTER_CELL_949ec677", "sort": {"node": "KSort", "name": "Int"}}}, "constraints": [{"node": "KApply", "label": {"node": "KLabel", "name": "#Equals", "params": [{"node": "KSort", "name": "Bool"}, {"node": "KSort", "name": "GeneratedTopCell"}]}, "args": [{"node": "KToken", "token": "false", "sort": {"node": "KSort", "name": "Bool"}}, {"node": "KApply", "label": {"node": "KLabel", "name": "LocalDeclCellMap:in_keys", "params": []}, "args": [{"node": "KApply", "label": {"node": "KLabel", "name": "<index>", "params": []}, "args": [{"node": "KToken", "token": "_3", "sort": {"node": "KSort", "name": "LocalToken"}}], "arity": 1, "variable": false}, {"node": "KVariable", "name": "LOCALDECLS_CELL_34597eca", "sort": {"node": "KSort", "name": "LocalDeclCellMap"}}], "arity": 2, "variable": false}], "arity": 2, "variable": false}, {"node": "KApply", "label": {"node": "KLabel", "name": "#Equals", "params": [{"node": "KSort", "name": "Bool"}, {"node": "KSort", "name": "GeneratedTopCell"}]}, "args": [{"node": "KToken", "token": "false", "sort": {"node": "KSort", "name": "Bool"}}, {"node": "KApply", "label": {"node": "KLabel", "name": "FunctionCellMap:in_keys", "params": []}, "args": [{"node": "KApply", "label": {"node": "KLabel", "name": "<fnKey>", "params": []}, "args": [{"node": "KVariable", "name": "FN_KEY", "sort": {"node": "KSort", "name": "FunctionLikeKey"}}], "arity": 1, "variable": false}, {"node": "KVariable", "name": "FUNCTIONS_CELL_fe1cac3f", "sort": {"node": "KSort", "name": "FunctionCellMap"}}], "arity": 2, "variable": false}], "arity": 2, "variable": false}]}}], "splits": [], "ndbranches": [], "vacuous": [], "stuck": [], "aliases": {}}
```

Comment:
Are these `.json` proof artefacts useful to store on the repo? I don't think it is a problem to include them necessarily, but they might go stale if we make changes in the future.

Reply:
It is for the purpose of demonstration and documentation at this stage.
Also, I wanted to keep it here and view the changes of proof functionality.
I will think about a better presentation in future.